File: Python/INICIANTE/1983 O Escolhido.py
#SUBPROGRAMA
#PROGRAMA PRINCIPAL
maior = 0
alunos = int(input())
while alunos != 0:
    matricula, nota = map(float, input().split())
    int(matricula)
    if maior < nota:
        maior = nota
        aprovado = matricula
    alunos -= 1
if maior < 8:
    print("Minimum note not reached")
else:
    print("%d" % aprovado)

File: C/INICIANTE/1983 O Escolhido.c
#include <stdio.h>

int main() {
    int alunos;
    long int matricula, aprovado;
    float nota, maior=0;
    scanf("%d", &alunos);
    while (alunos != 0)
    {
        scanf("%ld %f", &matricula, &nota);
        if (maior < nota)
        {
            maior = nota;
            aprovado = matricula;
        }
        alunos -= 1;
    }
    if (maior < 8)
        printf("Minimum note not reached\n");
    else
        printf("%ld\n", aprovado);
    return 0;
}
